omniauth-apple.gemspec

```
@@ -36,11 +36,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'omniauth-oauth2'
-  spec.add_dependency 'jwt'
+  spec.add_dependency "omniauth-oauth2", "~> 1.7.1"
```

Hi @lucas-aragno, thanks for creating this PR! 🙌🏼 

Is it possible to allow also [omniauth-oauth2 v1.8](https://github.com/omniauth/omniauth-oauth2/releases/tag/v1.8.0) like this?

```suggestion
  spec.add_dependency "omniauth-oauth2", "~> 1.7"
```

---

Yes, absolutely! that makes a lot sense. I'm adding the commit now.

---

Thank you! 😊